### PR TITLE
Changelog entry for JSON support in nr-assistant

### PR DESCRIPTION
**Pull request by Steve-Mcl**
## Description

Adds changelog entry for new JSON support

## Related Issue(s)

#2346

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [x] Documentation has been updated

### CODE REVIEW

**Comment by Steve-Mcl** [comment]
Not to be published until after nr-assistant 0.1.1 release is cut

File: src/changelog/2024/07/flowfuse-assistant-json.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+![Resulting JSON](./images/ask-assistant-json-1b.png)
+
+### Asking the assistant to generate JSON from within the `inject` node for the `payload` value 
+
+![Prompting from within the Inject Nodes Typed Input](./images/ask-assistant-json-2a.png)
+![Resulting JSON](./images/ask-assistant-json-2b.png)
+
+This is just the beginning of the FlowFuse Assistant's capabilities. Stay tuned!
+
+_JSON Support is available in the FlowFuse Assistant plugin starting from version 0.1.1_
```
Comment (knolleary) [inline src/changelog/2024/07/flowfuse-assistant-json.md:36]: Once we have it published, we should dbl check instructions on how to update the plugin. Keeping in mind that plugins won't show up in the palette manager in NR 3.x - that only arrived in 4.x

Comment (Steve-Mcl) [reply src/changelog/2024/07/flowfuse-assistant-json.md]: Good shout. Added update instructions for both streams

File: src/changelog/2024/07/flowfuse-assistant-json.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+### Asking the Assistant to generate JSON from within the `inject` node for the `payload` value 
+
+![Prompting from within the Inject Nodes Typed Input](./images/ask-assistant-json-2a.png)
+![Resulting JSON](./images/ask-assistant-json-2b.png)
+
+This is just the beginning of the FlowFuse Assistant's capabilities. Stay tuned!
+
+
+#### Updating the Assistant to get these new features
+If you are runing Node-RED V4.x, head over the the Palette Manager, update the plugin and restart your instance.
```
Comment (knolleary) [inline src/changelog/2024/07/flowfuse-assistant-json.md:38]:
```suggestion
If you are running Node-RED V4.x, head over the the Palette Manager, update the plugin and restart your instance.

```

**Review by knolleary — APPROVED** [review]

**Review by knolleary — APPROVED** [review]

**Pull request merged**